app/src/main/java/org/mozilla/fenix/perf/ThreadPenaltyDeathWithIgnoresListener.kt

```diff
@@ -35,7 +35,7 @@ class ThreadPenaltyDeathWithIgnoresListener(
         if (shouldViolationBeIgnored(violation)) {
             logger.debug("Ignoring StrictMode ThreadPolicy violation", violation)
         } else {
-            penaltyDeath(violation)
+            // penaltyDeath(violation)
```

Comment:
is this a temporary comment? should a comment be added to give some context why this was taken out?

Reply:
It's a local change for one of my test phones. I will remove it.

Reply:
Heh, I do that locally all the time too. We need to fix this thing. 😅